packages/app-builder/public/locales/fr/data.json

```
@@ -0,0 +1,66 @@
+{
+  "create_field.button_accept": "Créer un champ",
+  "create_field.description_placeholder": "Description du champ",
+  "create_field.is_enum.subtitle": "Il faut un nombre limité de valeurs distinctes, comme un code de statut",
+  "create_field.is_enum.title": "Ce champ est une valeur énumérée",
+  "create_field.name_placeholder": "Nom du champ",
+  "create_field.option_optional": "Facultatif",
+  "create_field.option_required": "Requis",
+  "create_field.required": "Requis",
+  "create_field.title": "Créer un nouveau champ",
+  "create_field.type": "Type",
+  "create_field.type_bool": "Booléen",
+  "create_field.type_float": "Nombre",
+  "create_field.type_string": "Texte",
+  "create_field.type_timestamp": "Horodatage",
```

```suggestion
  "create_field.type_timestamp": "Timestamp",
```
Je sais pas quelle est la meilleure traduction, mais je crois que je préfère encore l'anglais ici